components/actionbutton/stories/template.js

```diff
@@ -128,3 +128,80 @@ export const Template = ({
 		</button>
 	`;
 };
+
+export const ActionButtonsWithIconOptions = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+    ${Template({
+      ...args,
+			iconName: undefined,
+    }, context )}
+    ${Template({
+      ...args,
+    }, context )}
+    ${Template({
+      ...args,
+      hideLabel: true,
+    }, context )}
+	${Template({
+      ...args,
+      hideLabel: true,
+      hasPopup: "true",
+    }, context )}
+	${Template({
+      ...args,
+			iconName: undefined,
+      hasPopup: "true",
+    }, context )}
+  `,
+}, context );
```

Comment:
I'm not sure we need the `context` here, passed to `Container` (I think I did that wrong in my one comment from my last review about `context`). Just looking at your other `Container`s and the props for `Container()`, I don't think they have context passed.